### PR TITLE
fixed #24177 - doc inspectdb database view support

### PR DESCRIPTION
**Pull request by luzfcb**
Fix https://code.djangoproject.com/ticket/24177#ticket

### CODE REVIEW

File: docs/ref/django-admin.txt [hunk 1]
```diff
@@ -337,16 +337,16 @@ inspectdb
 
 .. django-admin:: inspectdb
 
-Introspects the database tables in the database pointed-to by the
+Introspects the database views and tables in the database pointed-to by the
```
Comment (MarkusH) [inline docs/ref/django-admin.txt]: "tables and views"

File: docs/ref/django-admin.txt [hunk 2]
```diff
@@ -337,16 +337,16 @@ inspectdb
...
 :setting:`NAME` setting and outputs a Django model module (a ``models.py``
 file) to standard output.
 
 Use this if you have a legacy database with which you'd like to use Django.
-The script will inspect the database and create a model for each table within
-it.
+The script will inspect the database and create a model for each table and views
```
Comment (MarkusH) [inline docs/ref/django-admin.txt]: "table or view within it."

File: docs/releases/1.8.txt [hunk 3]
```diff
@@ -415,6 +415,9 @@ Management Commands
   introspect :class:`~django.db.models.AutoField` for MySQL and PostgreSQL
   databases.
 
+* :djadmin:`inspectdb` now able to introspect ``database View`` for MySQL,
```
Comment (MarkusH) [inline docs/releases/1.8.txt]:
```
:djadmin:`inspectdb` now introspects database views on MySQL, Oracle, PostgreSQL and SQLite.
```

File: docs/howto/legacy-databases.txt [hunk 4]
```diff
@@ -64,6 +64,10 @@ If you do want to allow Django to manage the table's lifecycle, you'll need to
 change the :attr:`~django.db.models.Options.managed` option above to ``True``
 (or simply remove it because ``True`` is its default value).
 
+.. versionadded:: 1.8
+
+Feature to inspect database views was added to PostgreSQL, MySQL, Oracle and SQLite.
```
Comment (MarkusH) [inline docs/howto/legacy-databases.txt]: Could you sort the database backend in alphabetical order, please.

File: docs/ref/django-admin.txt [hunk 5]
```diff
@@ -390,6 +390,10 @@ it because ``True`` is its default value).
 The :djadminopt:`--database` option may be used to specify the
 database to introspect.
 
+.. versionadded:: 1.8
+
+Feature to inspect database views was added to PostgreSQL, MySQL, Oracle and SQLite.
```
Comment (MarkusH) [inline docs/ref/django-admin.txt]: Same as above, please.

**Comment by luzfcb** [comment]
@MarkusH done. Thanks for review.

Sorry for the mistakes, english is not my native language. I still have quite a lot to learn.

File: docs/howto/legacy-databases.txt [hunk 6]
```diff
@@ -64,6 +64,10 @@ If you do want to allow Django to manage the table's lifecycle, you'll need to
 change the :attr:`~django.db.models.Options.managed` option above to ``True``
 (or simply remove it because ``True`` is its default value).
 
+.. versionadded:: 1.8
+
+Feature to inspect database views was added to MySQL, Oracle, PostgreSQL and SQLite.
```
Comment (berkerpeksag) [inline docs/howto/legacy-databases.txt]: Could you use 4-space indentation?

```rst
.. versionadded:: 1.8

    Feature to inspect [...]
```

File: docs/ref/django-admin.txt [hunk 7]
```diff
@@ -390,6 +389,10 @@ it because ``True`` is its default value).
 The :djadminopt:`--database` option may be used to specify the
 database to introspect.
 
+.. versionadded:: 1.8
+
+Feature to inspect database views was added to MySQL, Oracle, PostgreSQL and SQLite.
```
Comment (berkerpeksag) [inline docs/ref/django-admin.txt]: Could you use 4-space indentation?

```rst
.. versionadded:: 1.8

    Feature to inspect [...]
```

**Comment by luzfcb** [comment]
@berkerpeksag done

**Comment by MarkusH** [comment]
Merged with minor changes in bd691f4586c8ad45bd059ff9d3621cbf8afdcdce. Thanks!

**Comment by timtan** [comment]
I am using django 1.11,  it seems that the feature is missing?

**Comment by luzfcb** [comment]
@timtan this commit was reverted, see why: https://code.djangoproject.com/ticket/25038#comment:1

in short, django does not support inspectdb from `database views` out of the box.

Related to tickets:
https://code.djangoproject.com/ticket/6148
https://code.djangoproject.com/ticket/7556
https://code.djangoproject.com/ticket/22673

**Pull request closed without merging**